src/test/java/com/github/krzysiekjodlowski/ox/BoardTest.java

```diff
@@ -4,63 +4,109 @@
 import org.testng.annotations.Test;
 
 import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
 public class BoardTest {
 
     @DataProvider
-    public static Object[][] examplesOfMovesStoredInBoard() {
+    private Object[][] sizesFrom3UpTo10WithRepresentations() {
```

Comment:
I am not sure but you can move data provider to other class. That can make this class easier to read.

Reply:
Cool idea, done!